src/server/html.js

```diff
@@ -22,6 +22,8 @@ export default class Html extends React.Component {
           <meta charSet="utf-8" />
           <meta content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" name="viewport" />
           <title>{this.props.title}</title>
+          <script src="//cdnjs.cloudflare.com/ajax/libs/nprogress/0.1.6/nprogress.min.js"></script>
```

Comment:
Never do that. Always use npm and one js build. Probability of saving from cdn is super small. Two requests sucks.